src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationService service;
+
+    @BeforeEach
+    void init() {
+        service = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void registerValidUser_Ok() {
+        User fitUser = new User("validLogin", "123456789", 25);
+        User actual = service.register(fitUser);
+        assertEquals(fitUser, actual);
+    }
+
+    @Test
+    void registerNullUser_notOk() {
+        User nullUser = null;
+        assertThrows(RuntimeException.class, () -> {
+            service.register(nullUser);
+        });
+    }
+
+    @Test
+    void registerUserWithTooShortPassword_notOk() {
+        User peter = new User("IvanGun", "1", 25);
```

Comment:
```suggestion
        User user = new User("IvanGun", "1", 25);
```
Check other places, make name understandable